scene/gui/code_edit.h

```diff
@@ -381,6 +391,16 @@ class CodeEdit : public TextEdit {
 
 	String get_auto_brace_completion_close_key(const String &p_open_key) const;
 
+	/* Code Actions */
+	void apply_text_edit(const ScriptLanguage::TextEditOperation &p_edit);
+	void apply_code_action(const ScriptLanguage::CodeActionOperation &p_code_action);
+	void clear_code_actions();
+	void add_code_actions(int p_line, const Vector<ScriptLanguage::CodeActionOperation> &p_code_actions);
+	void set_draw_code_actions(bool p_draw);
+	bool is_draw_code_actions_enabled() const;
+	HashMap<int, Vector<ScriptLanguage::CodeActionOperation>> code_actions;
+	int current_code_action_line; // The last-selected line to use code actions from.
```

Comment:
```suggestion
	int current_code_action_line = -1; // The last selected line to use code actions from.
```
Or some appropriate default